STABLE.md

```diff
@@ -2,10 +2,8 @@
 
 The following stable branches are kindly maintained by trusted volunteers:
 
--  4.14.x
+-  4.15.x
    -  git
-      -  [main](https://www.alejandro-colomar.es/src/alx/shadow/stable/shadow.git/log/?h=4.14.x)
```

Comment:
The main repository for the stable branch is still that one, github being a mirror.  You can s/14/15/ there.

Reply:
4.15.1 is a glitch in the 4.15.x series.  :)

Reply:
> The main repository for the stable branch is still that one, github being a mirror.

Ahh, just saw the certificate is self-signed. I mistakenly thought the main one is unmaintained!

Reply:
It's documented here:
<https://www.alejandro-colomar.es/ssl>  :)

Reply:
> It's documented here: https://www.alejandro-colomar.es/ssl :)

Such a good idea!